build/contracts/ConvertLib.json

```diff
@@ -25,15 +25,15 @@
       "type": "function"
     }
   ],
-  "bytecode": "0x6060604052341561000f57600080fd5b60b08061001d6000396000f300606060405260043610603f576000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff16806396e4ee3d146044575b600080fd5b606160048080359060200190919080359060200190919050506077565b6040518082815260200191505060405180910390f35b60008183029050929150505600a165627a7a72305820bd47c599c4d52b4df09fe111ac59c522d51e1845f6c609784a2893e317d4a3900029",
-  "deployedBytecode": "0x606060405260043610603f576000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff16806396e4ee3d146044575b600080fd5b606160048080359060200190919080359060200190919050506077565b6040518082815260200191505060405180910390f35b60008183029050929150505600a165627a7a72305820bd47c599c4d52b4df09fe111ac59c522d51e1845f6c609784a2893e317d4a3900029",
+  "bytecode": "0x6060604052341561000f57600080fd5b60b08061001d6000396000f300606060405260043610603f576000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff16806396e4ee3d146044575b600080fd5b606160048080359060200190919080359060200190919050506077565b6040518082815260200191505060405180910390f35b60008183029050929150505600a165627a7a72305820165484e7967c6650431ff0b29ce9f21d15c1d2103286f886017f2e36da6b067b0029",
```

Comment:
I think we should maybe gitignore this file, like you suggested. I need to google..

Reply:
yeah I should have never committed the files in `/contracts` because they are generated by the build. I will have to go delete them from develop

Reply:
I agree on this, all the build directory can be ignored and we need to compile the contracts at each sync